packages/fxa-payments-server/src/routes/Subscriptions/Cancel/CancelSubscriptionPanel.test.tsx

```diff
@@ -0,0 +1,212 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file replaces the rendering tests using react-test-renderer that were in `src/routes/Subscriptions/PaymentUpdateForm.test.tsx`, as well as adding a couple event handling tests that were previously missing.